edx2bigquery/rephrase_tracking_logs.py

```diff
@@ -278,6 +278,25 @@ def check_empty(data, *keys):
 
     data.pop('event_js')	# leftover from mongo import script
 
+    #-----------------------------------------
+    # check for null values in speed_change_video
+    # Error encountered parsing LAC data from Oct. 2013
+
+    def check_empty_value(data_dict, *keys):
+        key = keys[0]
+        if type(data_dict)==dict and key in data_dict:
+            if len(keys)==1:
+                if float(data_dict[key]) != float(data_dict[key]):
```

Comment:
This conditional does not make sense, and will cause an exception if data_dict[key] cannot be converted to a float.